src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -53,29 +53,34 @@ def get_attrs(
         TODO(dcramer): assert on relations
         """
 
-        # Preload to avoid fetching each user individually
+        # Preload to avoid fetching each team individually
         prefetch_related_objects(
             item_list,
-            "user",
```

Comment:
I removed this as the relation between organizationmember and user is going away.